src/App.jsx

```diff
@@ -1,42 +1,55 @@
 import React, { useState } from "react";
-import "aframe";
-import "aframe-event-set-component";
-import "aframe-orbit-controls";
 
-import "./Aframe/loader.js";
-import "./Aframe/buttons-check.js";
-import "./Aframe/render-2d-clipplane";
-import "./Aframe/cursor-listener";
-
-import "./styles/main.scss";
-
-import { SelectorProvider } from "./context/selector-context.js";
+import config from "./assets/config.json";
+import haline from "./assets/colormaps/haline.png";
+import thermal from "./assets/colormaps/thermal.png";
 
 import Header from "./components/Header";
 import InfoText from "./components/InfoText";
 import ModelSelector from "./components/ModelSelector";
+import VolumeViewerWrapper from "./components/VolumeViewerWrapper";
 import Instructions from "./components/instructions/Instructions";
 import Footer from "./components/Footer";
-import VolumeViewerWrapper from "./components/volume-viewer/VolumeViewerWrapper.jsx";
-import { ControlsProvider } from "./context/controls-context.js";
+import "./styles/main.scss";
+
+const ALL_COLOR_MAPS = {
+  Haline: haline,
+  Thermal: thermal,
+};
+const BUTTONS = {
+  season: config.season,
+  tide: config.tide,
+  measurement: config.measurement,
+};
 
 export default function App() {
   const [controlsVisible, setControlsVisible] = useState(false);
 
+  const [selection, setSelection] = useState({
+    season: BUTTONS.season[0],
+    tide: BUTTONS.tide[0],
+    measurement: BUTTONS.measurement[0],
+  });
```

Comment:
```suggestion
  const [selection, setSelection] = useState(Object.keys(BUTTONS).reduce((obj, key) => ({...obj, [key]: BUTTONS[key][0]}), {});
```
A bit less readable, but it's one less place to update if the variables change.   Might be worth making the variables their own `const` then also using that to construct `BUTTONS` (would also obviate the need for the `Object.keys` call here.

Reply:
Hmm.. maybe make it better in the config.json?

Reply:
I decided to leave it as `Object.keys` but used the same logic in `ModelSelector.jsx` to generate the button groups